src/main/java/uk/gov/hmcts/juror/api/moj/repository/jurorresponse/IJurorDigitalResponseRepositoryMod.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.juror.api.moj.repository.jurorresponse;
+
+import com.querydsl.core.Tuple;
+
+import java.util.List;
+
+public interface IJurorDigitalResponseRepositoryMod {
+    Tuple getAssignRepliesStatistics();
+
+    List<Tuple> getAssignRepliesStatisticForUsers();
```

Comment:
Returning a Tuple should be avoided if possible (Java objects make code more readable)

Reply:
Would normally avoid but in this case as the query is generic it made sense to use a Tuple